lib/data_update_scripts/20230517132257_populate_suggested_tags_from_settings.rb

```diff
@@ -0,0 +1,17 @@
+module DataUpdateScripts
+  class PopulateSuggestedTagsFromSettings
+    cattr_accessor :suggested_tags
+
+    def run(suggested_tags: settings_suggested_tags)
+      return unless suggested_tags.any?
+
+      Tag.where(name: suggested_tags).update_all suggested: true
+    end
+
+    private
+
+    def settings_suggested_tags
+      suggested_tags || Settings::General.suggested_tags
```

Comment:
Did you mean to put a `||=` here?

Reply:
It's `||` on purpose, but it's a bit dense — there's a class-level attr_acessor `suggested_tags`, which allows the test to set up these tags, the `cattr_accessor` macro also creates instance-level methods, so this will be `class' suggested_tags or else Settings suggested_tags`.